libcextract/EnumConstTbl.cpp

```diff
@@ -20,7 +20,7 @@ EnumConstantTable::EnumConstantTable(ASTUnit *ast)
 {
   clang::ASTUnit::top_level_iterator it;
   for (it = ast->top_level_begin(); it != ast->top_level_end(); ++it) {
-    EnumDecl *decl = dynamic_cast<EnumDecl *>(*it);
+    EnumDecl *decl = dyn_cast<EnumDecl *>(*it);
```

Comment:
Please take a look at other locations where `dyn_cast` takes place. If you look there you will see that it does not accept a pointer as argument in the `<>` region but rather the type of the class itself.

Please compile your changes before submitting.